packages/dev/core/src/Gizmos/positionGizmo.ts

```diff
@@ -208,6 +209,19 @@ export class PositionGizmo extends Gizmo implements IPositionGizmo {
         return this._planarGizmoEnabled;
     }
 
+    /**
+     * posture that the gizmo will be display
+     * When set null, default value will be used (Quaternion(0, 0, 0, 1))
+     */
+    public set customRotationQuaternion(customRotationQuaternion: Nullable<Quaternion>) {

```

Comment:
In fact, you will have to reimplement the getter in each class, because [you can't override only the setter](https://github.com/babel/babel/issues/2449). The getter must also be reimplemented, else you will get `undefined` when accessing it from the derived class:
```javascript
class A { get value() { console.log("get value in A"); } set value(v) { console.log("set value in A"); } }
class A2 extends A { set value(v) { console.log("set value in A2"); } }
a=new A(); a.value;
a2=new A2(); a2.value;
```
result:
![image](https://github.com/BabylonJS/Babylon.js/assets/4152247/016701fa-fe39-4f32-be41-4b9206d1ffb2)

Reply:
setters back in